external_review/export_DO.sh

```diff
@@ -5,8 +5,13 @@ function export_DO {
     echo $2
     geography_level=$1
     category=$2
-    filename="${category}_${geography_level}.csv"
-    SPACES="spaces/edm-publishing/db-eddt/${category}"
+    local filename="${category}_${geography_level}.csv"
+    local branchname=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD)
+    if [ "$branchname" = "main" ] ; then
+        SPACES="spaces/edm-publishing/db-eddt/${category}"
+    else
+        SPACES="spaces/edm-publishing/db-eddt/${branchname}/${category}"
+    fi
     mc cp .staging/$category/$filename $SPACES/$filename
     mc cp .staging/$category/metadata.yml $SPACES/metadata.yml
```

Comment:
Can we talk again about why not to just export all of `.staging/$category`? I think that would be easier to manage

Reply:
yeah I think my rationale for it is this eliminate the chances something unexpected ended up DO. It wasn't my design but I like it.

Reply:
Ok word. I think something unexpected ending up in DO is better than something we output not ending up there. This is why all the other pipelines are written to export everything I think. But it's ok either way I think we should run the action